docs/README.md

```diff
@@ -25,18 +25,17 @@ e.g. Kubernetes version, cloud provider, cluster type, node instance type.
 - *Profile*: a grouping of configuration bundles which will be installed into the cluster
 - *Configuration bundle*: a unit of configuration which contains (or references) one or
 more Kubernetes manifests. A bundle can encapsulate anything that can be deployed onto a cluster:
-an RBAC ruleset, an add-on, an application, etc... 
+an RBAC ruleset, an add-on, an application, etc...
 
 ## Arlon Benefits
 
 - Improves time to market by enabling better velocity for developers through infrastructure management that is more fluid and agile. Define, store, change and enforce your cluster infrastructure & application add-ons at scale.  
-- Reduces the risk of unexpected infrastructure downtime and outages, or unexpected security misconfiguration, with consistent management of infrastructure and security policies.   
-- Allows IT and Platform Ops admins to operate large scale of clusters, infrastructure & add-ons with significantly reduced team size & operational overhead, using GitOps. 
+- Reduces the risk of unexpected infrastructure downtime and outages, or unexpected security misconfiguration, with consistent management of infrastructure and security policies.
+- Allows IT and Platform Ops admins to operate large scale of clusters, infrastructure & add-ons with significantly reduced team size & operational overhead, using GitOps.
 
 ## Contents
 
-- [Concepts](./concepts.md) 
+- [Concepts](./concepts.md)
```

Comment:
I see that details about clusterspec still exists in concepts.md file. we should remove the clusterspec details in that file as well right?

Reply:
Done, thanks.